src/CollapsibleNav.js

```diff
@@ -48,13 +46,12 @@ const specCollapsibleNav = {
     /*
      * this.props.collapsible is set in NavBar when an eventKey is supplied.
      */
-    const collapsible = this.props.collapsible || this.props.collapsable;
-    let classes = collapsible ? this.getCollapsibleClassSet('navbar-collapse') : null;
+    const classes = this.props.collapsible ? this.getCollapsibleClassSet('navbar-collapse') : null;
 
     return (
       <div eventKey={this.props.eventKey} className={classNames(this.props.className, classes)} >
         {ValidComponentChildren.map(this.props.children,
-          collapsible ?
+          this.props.collapsible ?
           this.renderCollapsibleNavChildren :
           this.renderChildren
```

Comment:
Maybe this ternary should be moved outside of this `map` call? This setup is pretty rough.